benchmarks/decoders/benchmark_decoders_library.py

```diff
@@ -535,6 +595,8 @@ def run_benchmarks(
     results = []
     df_data = []
     verbose = False
+    # TODO: change this back before landing.
+    min_runtime_seconds = 0.1
```

Comment:
I have an unmerged change in PR #362 to make this sort of benchmark testing easier: https://github.com/pytorch/torchcodec/pull/362/files#diff-c378bd5d03e7daa116eaeeeb86921e8134f0feefb0fcdf9f020f872676c5c00dR31-R34